ecl/regress/issue14052.ecl

```diff
@@ -0,0 +1,54 @@
+layout1 := {
```

Comment:
These code looks familiar, (regarding the comments) is it cannibalised (just curious)?